src/pages/index.js

```diff
@@ -66,20 +108,85 @@ function renderCard(data) {
 /*-------------------------------------------------------------------------------*/
 
 function handleProfileFormSubmit(data) {
+  profileEditPopup.setLoading(true);
   userInfo.setUserInfo(data);
+  api.updateUserInfo(userInfo.getUserInfo())
+  .catch((err) => {
+    console.error(err);
+  })
+  .finally(() => profileEditPopup.setLoading(false));
   profileFormValidator.resetValidation();
```

Comment:
`profileFormValidator.resetValidation()` is not needed after submission


`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs.